pkgs/development/tools/profiling/pprof/default.nix

```diff
@@ -5,20 +5,19 @@
 
 buildGoModule rec {
   pname = "pprof";
-  version = "unstable-2024-02-27";
+  version = "unstable-2024-05-09";
```

Comment:
This doesn't follow the unstable version guideline anymore: https://github.com/NixOS/nixpkgs/blob/87f9e7dd92cd063a5a27a009d0fee25c84f16c83/pkgs/README.md#package-naming

Reply:
Fixed. Added the missing `0-` prefix.